features/tabs.json

```diff
@@ -0,0 +1,7 @@
+{
+    "_meta": {
+        "description": "A feature flag that controls the subfeatures related the tabs.",
+        "sampleExcludeRecords": {}
+    },
+    "exceptions": []
```

Comment:
Maybe drop `exceptions` and `sampleExcludeRecords` as I don't think we want them for this feature.